audb/core/utils.py

```diff
@@ -106,3 +108,15 @@ def _lookup(
         f'for database '
         f"'{name}'."
     )
+
+
+def _mkdir_tree(
```

Comment:
```suggestion
def mkdir_tree(
```
as we want to import it in other files.

Reply:
done